clusters/ztp-policies/common/group-policies/group-dellr740xd-vse3.yaml

```diff
@@ -0,0 +1,200 @@
+    # The setting below overrides the default "worker" selector predefined in
+    # the source-crs. The change is recommended on SNOs configured with PTP
+    # event notification for forward compatibility with possible SNO expansion.
+    # When the default setting is left intact, then in case of an SNO
+    # expansion with one or more workers, PTP operator
+    # would not create linuxptp-daemon containers on the worker node(s). Any
+    # attempt to change the daemonsetNodeSelector will result in ptp daemon
+    # restart and time synchronization loss.
+    # After complying with the policy, complianceType can be set to a safer "musthave"
+    # - fileName: PtpOperatorConfigForEvent.yaml
+    #   policyName: "config-policy"
+    #   complianceType: mustonlyhave
+    #   spec:
+    #     daemonNodeSelector:
+    #       node-role.kubernetes.io/worker: ""
+    #     ptpEventConfig:
+    #       storageType: "storage-class-http-events"
+
+#    - fileName: PtpConfigSlave.yaml   # Change to PtpConfigSlaveCvl.yaml for ColumbiaVille NIC
+#      policyName: "config-policy"
+#      metadata:
+#        name: "du-ptp-slave"
+#      spec:
+#        profile:
+#        - name: "slave"
+          # This interface must match the hardware in this group
+#          interface: "ens5f0"
+#          ptp4lOpts: "-2 -s --summary_interval -4"
+#          phc2sysOpts: "-a -r -n 24"
+    - fileName: SriovOperatorConfig.yaml
+      policyName: "config-policy"
+      # For existing clusters with node selector set as "master",
+      # change the complianceType to "mustonlyhave".
+      # After complying with the policy, the complianceType can
+      # be reverted to "musthave"
+      complianceType: musthave
+      spec:
+        configDaemonNodeSelector:
+          node-role.kubernetes.io/worker: ""
+#    - fileName: StorageLV.yaml
+#      policyName: "config-policy"
+#      spec:
+#        storageClassDevices:
+#        - storageClassName: "example-storage-class-1"
+#          volumeMode: Filesystem
+#          fsType: xfs
+#          devicePaths:
+#          - /dev/sdb1
+#        - storageClassName: "example-storage-class-2"
+#          volumeMode: Filesystem
+#          fsType: xfs
+#          devicePaths:
+#          - /dev/sdb2
+        # This is required if PTP and BMER operators use HTTP transport.
+        # The disk labels are created by ignitionConfigOverride in siteConfig.
+        # - storageClassName: "storage-class-http-events"
+        #   volumeMode: Filesystem
+        #   fsType: xfs
+        #   devicePaths:
+        #   - /dev/disk/by-partlabel/httpevent1
+        #   - /dev/disk/by-partlabel/httpevent2
+    - fileName: DisableSnoNetworkDiag.yaml
+      policyName: "config-policy"
+    - fileName: PerformanceProfile.yaml
+      policyName: "config-policy"
+      spec:
+        cpu:
+          # These must be tailored for the specific hardware platform
+          isolated: "2-39,42-79"
+          reserved: "0-1,40-41"
```

Comment:
```suggestion
          isolated: "1,3-39,41,43-79"
          reserved: "0,2,40,42"
```

Reply:
Suggest you align the reserved CPUs to the NUMA 0 topology, since this is a Dual Socket based system.